src/cli/Cli.ts

```diff
@@ -947,6 +947,23 @@ export class Cli {
     return response;
   }
 
+  public static async interactivePrompt(promptMessage: string, errorMessage: string, values: { [key: string]: object }): Promise<object | CommandError> {
```

Comment:
the name `interactivePrompt` is a bit tricky and mysterious 😉. Also if the `interactive` setting is false (like the default) we don't prompt at all. So in this method we actually 'handle multiple values result' in a way that if the `interactive` is 'off' we present the error and if it's 'on' we prompt. 

What do you think we rename it to `handleMultipleResultsFound` or something similar? Any better suggestions?